### ISSUE TITLE
One keyboard shortcut for review modes

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3392]
**Reported by ondrosik on 2013-07-31 18:39**
Currently, we are using nvda+page down and page up for switching between rewiev modes. My idea is as follows: Use only one shortcut which will cycle between available modes. There are just three and sometimes only two available modes so cycling is not so time consuming. In addition, NVDA will not tell us "no next rewiev mode" and comfuse user to remember whichone was the lastone. We will also get one free binding.

**Comment by bhavyashah** [comment]
Given that our concerns about consumption of keyboard shortcuts by NVDA has grown with NVDA's evolving feature set and functionality over time, and since @jcsteh did mention in 2013 that this could be reconsidered in the future, I would like to request discussion on the same while we are in the year 2017. As an NVDA user who started using NVDA in a big way post the flat review era, plus the fact that it has been several years that NVDA has adopted the more modern review modes approach as opposed to flat review, I doubt whether backwards compatibility would even be a worry. I agree with @josephsl that NVDA+PgUp/PgDn should be scrapped and something more generic and singular like NVDA+/ should be used. @leonardder @derekriemer @Qchristensen Thoughts?

**Comment by derekriemer** [comment]
I see no reason from a usability point why we shouldn't do this.

**Comment by derekriemer** [comment]
Closing in favor of #3958 because it's much more abstract. @feerrenrut do you know who should prioritize this?

**Comment by bhavyashah** [comment]
Did you close this ticket accidentally? You just closed another
related ticket too, Derek. Feel free to undo anything unintentional,
either by reopening this one or its counterpart.

On 7/31/17, Derek Riemer <notifications@github.com> wrote:
> Closed #3392.
>
> --
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/3392#event-1185510385


-- 
Best Regards
Bhavya Shah

Blogger at Hiking Across Horizons: https://bhavyashah125.wordpress.com/

Contacting Me
E-mail Address: bhavya.shah125@gmail.com
Follow me on Twitter @BhavyaShah125 or www.twitter.com/BhavyaShah125
Mobile Number: +91 7506221750

**Comment by derekriemer** [comment]
Nope, I need to mark it as a duplicate though. I have a comment explainin why I closed it, because #3908 is more abstract and general.

**Comment by bhavyashah** [comment]
Figured. BTW, #3908 is not what you meant in your last comment. Feel free to edit.

**Comment by derekriemer** [comment]
[**@bhavyashah**](https://github.com/bhavyashah) commented on [Jul 31, 2017, 9:00 PM MDT](https://github.com/nvaccess/nvda/issues/3392#issuecomment-319254413 "2017-08-01T03:00:44Z - Replied by Github Reply Comments"):
> Figured. BTW, #3908 is not what you meant in your last comment. Feel free to edit.

Nice catch, thanks!
